adapters/emx_digital/emx_digital.go

```diff
@@ -137,6 +137,41 @@ func buildImpBanner(imp *openrtb.Imp) error {
 	return nil
 }
 
+func buildImpVideo(imp *openrtb.Imp) error {
+	imp.Ext = nil
```

Comment:
Just curious why the Ext is removed here.

Reply:
Team had removed it as we were not supporting it for display either - now adapter supports for both display and video for future use